### PR TITLE
Restructure README.md file

### LINKED ISSUE #2: Add documentation about the rendering Process
Headless - Rendering Process

### PR DESCRIPTION
**Pull request by nnatter**
Have a look at the result at: https://github.com/nnatter/SuluHeadlessBundle/tree/enhancement/readme

Fixes #2

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2018 MassiveArt Webservices GmbH
```
Comment (alexander-schranz) [inline LICENSE]: false

File: LICENSE [hunk 2]
```diff
@@ -0,0 +1,21 @@
+MIT License
```
Comment (alexander-schranz) [inline LICENSE]: Should also be changed in the `composer.json`

Comment (nnatter) [reply LICENSE]: done

File: README.md [hunk 3]
```diff
@@ -1,81 +1,226 @@
...
+   "template": "headless-template",
+   "content": {
+      "title": "Headless Example Page",
+      "url": "/headless-example",
+      "contacts": [
+         {
+            "id": 416,
+            "firstName": "Homer",
+            "lastName": "Simpson",
+            "fullName": "Home Simpson",
```
Comment (wachterjohannes) [inline README.md]: please write him correcty 😂  `Homer Simpson`

File: README.md [hunk 4]
```diff
@@ -1,81 +1,226 @@
...
+            "title": "Dr. ",
+            "position": "Executive Vice President"
```
Comment (wachterjohannes) [inline README.md]: his position is `Nuclear safety Inspector at the Springfield Nuclear Power Plan` :)

**Comment by alexander-schranz** [comment]
@nnatter missing the `babe.config.js`:

```
module.exports = {
    presets: ['@babel/env', '@babel/react'],
    plugins: [
        ['@babel/plugin-proposal-decorators', {'legacy': true}],
        ['@babel/plugin-proposal-class-properties', {'loose': true}]
    ]
};
```

File: README.md [hunk 5]
```diff
@@ -1,81 +1,226 @@
...
+```twig
+{% block content %}
+    <div id="sulu-headless-container"></div>
+    
+    {# initialize application with json data of current page to prevent second request on first load #}
+    <script>window.SULU_HEADLESS_VIEW_DATA = {{ jsonData|raw }};</script>
+    <script>window.SULU_HEADLESS_API_ENDPOINT = '{{ sulu_content_path('/api') }}';</script>
+    
+    {# start the single page application #}
+    <script src="/build/website/js/index.js"></script
```
Comment (alexander-schranz) [inline README.md]: we need atleast a tag `<body>` around this else if you just copy & paste the example don't work

File: README.md [hunk 6]
```diff
@@ -1,81 +1,226 @@
...
+  "name": "my-frontend-application",
   "main": "src/index.js",
   "private": true,
   "scripts": {
-    "build": "webpack src/index.js -o ../../public/build/headless/js/index.js --module-bind js=babel-loader -p --display-modules --sort-modules-by size",
-    "watch": "webpack src/index.js -w -o ../../public/build/headless/js/index.js  --module-bind js=babel-loader --mode=development --devtool source-map",
-    "lint": "eslint src",
-    "lint:fix": "eslint src --fix",
-    "depcruise": "depcruise src -c dependency-cruiser.json"
+    "build": "webpack src/index.js -o ../../public/build/website/js/index.js --module-bind js=babel-loader -p --display-modules --sort-modules-by size",
```
Comment (alexander-schranz) [inline README.md]: I would personally prefer building to `build/headless`

File: README.md (hunk 5 above)
Comment (alexander-schranz) [reply README.md]: I would use `build/headless`

File: README.md [hunk 7]
```diff
@@ -1,81 +1,226 @@
...
+    
+    {# start the single page application #}
+    <script src="/build/website/js/index.js"></script
+{% endblock %}
+```
+
+Additionally, you need to add the following files to your project to setup the single page application:
+
+<details>
+<summary>assets/website/package.json</summary>
```
Comment (alexander-schranz) [inline README.md]: move to `assets/headless`

**Pull request merged**